src/coprocessor/dag/executor/aggregation.rs

```diff
@@ -241,7 +242,7 @@ impl Executor for StreamAggExecutor {
         // If there is no data in the t, then whether there is 'group by' that can affect the result.
         // e.g. select count(*) from t. Result is 0.
         // e.g. select count(*) from t group by c. Result is empty.
-        if self.count == 0 && !self.group_by_exprs.is_empty() {
+        if !self.has_data && !self.group_by_exprs.is_empty() {
```

Comment:
Any test cases to cover `self.has_data=false`?

Reply:
These test cases in TiDB.
Now I add it here.